### ISSUE TITLE
Bento archives more issues than it found

### ISSUE THREAD

**Issue by arashd** [returntocorp/bento#34]
Ran `bento check` on the simple integration test and got the following:

```bash
$ bento check
⚠ 5 findings in 4.71 s
To suppress all findings run `bento archive`.

$ bento archive
✔ Rewrote the whitelist with 6 findings from 3 tools.
```

Not sure where the 5 found vs 6 archived discrepancy is coming from.

**Comment by tnarla** [comment]
I can't reproduce this, did something change since 3 days ago?

<img width="655" alt="Screen Shot 2019-10-07 at 4 28 26 PM" src="https://user-images.githubusercontent.com/5661037/66349900-8835dc80-e91f-11e9-88ee-0b0001761eaa.png">

**Comment by nbrahms** [comment]
I reproduced using `0.1.0r2`:

```(bento-core) ➜  simple git:(release-0.1.0) ✗ bento check
Running Bento checks...
  r2c.bandit: |███████████████| 00:01, 🍱
  r2c.eslint: |███████████████| 00:01, 🍱
  r2c.flake8: |███████████████| 00:02, 🍱

bar.py
  1:0  advice  Consider possible security implications associated with subprocess module. r2c.bandit     blacklist
  5:0  error   subprocess call with shell=True identified, security issue.                r2c.bandit     subprocess_popen_with_shell_equals_true

foo.py
  0:0  error   syntax error while parsing AST from file                                   r2c.bandit     error
  6:5  error   unexpected indentation                                                     r2c.flake8     E113

init.js
  1:1  warning Unexpected console statement.                                              r2c.eslint     no-console

⚠ 5 findings in 2.25 s

To suppress all findings run `bento archive`.
(bento-core) ➜  simple git:(release-0.1.0) ✗ bento archive
✔ Rewrote the whitelist with 6 findings from 3 tools.```

**Comment by nbrahms** [comment]
It looks like, specifically, the syntax error from flake8 is not getting displayed in the screen output.

**Comment by nbrahms** [comment]
👋 @arashd ... this should be fixed in 0.1.0r4. Happy hacking!

**Comment by dlukeomalley** [comment]
With the updates to the archive output, this appears to be fixed! Closing out.
